core/domain/contribution_stats_services_test.py

```diff
@@ -378,7 +378,8 @@ def test_get_translation_submitter_admin_stats(self) -> None:
             language_code='es',
             sort_by=None,
             topic_ids=None,
-            max_days_since_last_activity=90
+            max_days_since_start_date_for_last_activity=None,
```

Comment:
Why do we pass None here? Specially when we pass a value for `max_days_since_end_date_for_last_activity`

Reply:
Whenever we do not pass start_date_for_last_activity from frontend, None is passed to `max_days_since_start_date_for_last_activity` here. Also, backend APIs do not require us to pass a value for both `max_days_since_start_date_for_last_activity` and `max_days_since_end_date_for_last_activity`. Here I pased None to `max_days_since_start_date_for_last_activity` because in the models created earlier for these tests do not specify `first_contribution_date` using timedelta, hence it is difficult to figure out a specific value for `max_days_since_start_date_for_last_activity`.
All cases of filtering using both `max_days_since_start_date_for_last_activity` and `max_days_since_end_date_for_last_activity` have been covered in `core/controllers/contributor_dashboard_admin_test.py` file.